source/agora/test/Base.d

```diff
@@ -515,7 +510,11 @@ private mixin template TestNodeMixin ()
     /// Prints out the log contents for this node
     public void printLog ()
     {
+        import std.stdio;
```

Comment:
Might as well move it at module scope ? It's a unittest module, we don't really shy on the dependencies.